lib/melodica_inventory_web/views/item_view.ex

```diff
@@ -14,6 +14,18 @@ defmodule MelodicaInventoryWeb.ItemView do
   end
 
   def cover_url(%Item{images: images}) do
-    Url.for(hd(images).public_id, %{width: 600, height: 600, crop: "limit"})
+    Enum.map(images, fn(image)  ->
+      Url.for(image.public_id, %{width: 400, height: 300, crop: "limit"})
+    end)
+    |> Enum.with_index
+  end
```

Comment:
Isn't this duplicate of https://github.com/valo/melodica-inventory/pull/37/files#diff-fe15ea94e3e48f9f1558202d2b44156dR11?